server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestAddVotingConfigExclusionAction.java

```diff
@@ -55,9 +59,20 @@ protected RestChannelConsumer prepareRequest(final RestRequest request, final No
     }
 
     AddVotingConfigExclusionsRequest resolveVotingConfigExclusionsRequest(final RestRequest request) {
-        String nodeName = request.param("node_name");
+        String nodeDescriptions;
+
+        // TODO This request param is being deprecated
+        if (request.hasParam("node_name")) {
+            nodeDescriptions = request.param("node_name");
```

Comment:
We should throw an `IllegalArgumentException` if this is set as well as the query parameters, and should also throw an `IllegalArgumentException` if both query parameters are in use.

Reply:
Done.